controllers/aggregated_clusteroperator_controller.go

```diff
@@ -90,6 +90,7 @@ func (a *AggregatedClusterOperatorReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, nil
 	}
 	coBuilder.WithAvailable(openshiftconfigv1.ConditionTrue, "All POs in a successful state", "POsHealthy")
+	coBuilder.WithProgressing(openshiftconfigv1.ConditionFalse, "All POs in a successful state")
```

Comment:
We should probably have a follow-up to turn PO into PlatformOperators. Not blocking though.

Reply:
Yeah, agreed. We should also move these to constant variables so there's less potential for skew between the implementation and test code.